style.css

```diff
@@ -509,8 +518,131 @@ nav.active {
   width: 100%;
 }
 
-.random-projects:hover {
-  transform: scale(1.1);
+/* Window Modal Style */
+.overlay {
+  position: fixed;
+  width: 100%;
+  height: 100%;
+  max-width: inherit;
+  max-height: inherit;
+  background-color: rgba(0, 0, 0, 0.4);
+  -webkit-backdrop-filter: blur(10px);
+  backdrop-filter: blur(10px);
+  z-index: 5;
+  display: flex;
+  padding: 0 1.6rem;
+  justify-content: center;
+}
+
+.modalWindow {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  gap: 2.4rem;
+  width: max-content;
+  height: 90vh;
+  max-height: max-content;
+  margin: auto;
+  padding: 1.6rem;
+  background-color: white;
+  z-index: 5;
+  overflow-y: scroll;
+}
```

Comment:
- [ ] Your mobile popup cannot contain the entire content and still has a scrollbar resulting in a bad experience for the user. Take a deep look into the font sizes, paddings, margins and all the dimensions in the Figma design and try to replicate that on your project.
![Untitled](https://user-images.githubusercontent.com/61919617/181270774-0a8392f8-c685-47e3-91c6-540932a1d939.gif)